### DISCUSSION TITLE
Plan for Tasks management(Scrum/Kanban) support in outline

### DISCUSSION THREAD

**Discussion by nuhbye** [outline/outline#1312] [General] [answered]
Probably this might have been discussed in the community, but could not find any reference to the discussion. 

I wanted to check if there are any plans for adding task management(Kanban/Scrum) support to outline. I myself would like to support if needed but wanted to check what others think about such feature in outline.

**Comment by tommoor** [comment, marked as answer]
Howdy, what kind of usecase do you have in mind? I could see a world where we add some simple ability to assign tasks in the future but nothing as advanced as scrum/kanban. There is a lot of work to do on document management and workflows still.

I'd recommend taking a look at http://linear.app

**Reply by janbaykara** [reply]
My co-op uses Notion primarily so that we can write notes and things and link them to the tasks we're working on, and in our tasks, to write rich text things that link out to our other notes.

What we like is how you can view lists of documents in a few different ways, including a simple 'board' view where you organise cards by custom properties like "status" (e.g. In Progress).

Is this on the roadmap?

**Reply by bjornfiles** [reply]
I would love this too. I'd rather use Outline over Notion but my team needs basic Kanban-style card support to do so—even if it was just the look and feel of drag-and-drop cards and columns without more advanced task management etc. on day one.

**Comment by nuhbye** [comment]
Thanks for reply Tom. I had simple task planning and management in my mind majorly using boards and lanes. I will check the linear app.

**Comment by matbrgz** [comment]
For project manage I recommend Twake.app or ClickUp.com
